### ISSUE TITLE
Livetickeruhr (Spielticker/Statistik)

### ISSUE THREAD

**Issue by entaxi57** [digedag/cfc_league_fe#54]
Hallo René,

gibt es eine Möglichkeit, die Uhr beim Liveticker zu stoppen, ohne das sie auf null springt?

Beim Handball wird die Zeit öfters angehalten und wieder gestartet.

**Comment by digedag** [comment]
Nein, das ist leider nicht vorgesehen. Diese kurzen Pausen gibt es beim Fußball leider nicht. Das ist auch nicht ganz so trivial einzubauen, weil die Zeit auch serverseitig gespeichert wird. Die Uhr läuft ja auch weiter, wenn du den Ticker verlässt.

**Comment by entaxi57** [comment]
Hallo René,

schade. Da kann man nichts machen.

**Comment by entaxi57** [comment]
Hallo René,

schade. Da kann man nichts machen.

**Comment by digedag** [comment]
Ein Pause-Button ist jetzt drin. War gar nicht so einfach... :wink:

**Comment by entaxi57** [comment]
Hallo René,

super, danke.👍

Werde es schnellstens testen.

**Comment by entaxi57** [comment]
Hallo René,

ich habe gerade die neueste Version auf meiner Testumgebung installiert.

Jetzt bekommen ich im Backend beim Aufruf von T3Sports folgende Fehlermeldung:

DBAL fatal error: No handler found in handler_getFromTableList() for: "tx_cfcleague_games INNER JOIN tx_cfcleague_teams t1 ON (home= t1.uid) INNER JOIN tx_cfcleague_teams t2 ON (guest= t2.uid) " (SQL engine parse ERROR: No join field found in parseFromTables()!: near "(home= t1.uid) INNER JOIN tx_cfcleague_teams t2 ON")

**Comment by digedag** [comment]
Das klingt danach, als ob die DBAL-Extension installiert ist. Nur gehe ich mal davon aus, daß du nur T3sports aktualisiert hast. Ich weiß nicht, wie alt deine vorherige Version war. Am besten auch rn_base mit aktualisieren. Und dann den System-Cache leeren.

**Comment by entaxi57** [comment]
Hallo René,

ich habe jetzt ebenfalls die rn_base aktualisiert. Leider funktioniert es immer noch nicht.

Sobald ich "Spielticker/Statistik" bei einem vorhandenen Spielplan aufrufe, bekomme ich folgende Fehlermeldung:
DBAL fatal error: No handler found in handler_getFromTableList() for: "tx_cfcleague_games INNER JOIN tx_cfcleague_teams t1 ON (home= t1.uid) INNER JOIN tx_cfcleague_teams t2 ON (guest= t2.uid) " (SQL engine parse ERROR: No join field found in parseFromTables()!: near "(home= t1.uid) INNER JOIN tx_cfcleague_teams t2 ON")

Ich habe dann den alten Spielplan gelöscht und einen neuen erzeugt. Gleiches Problem.

Wenn ich "Person suchen" aufrufe bekomme ich diese Fehlermeldung:
DBAL fatal error: No handler found in handler_getFromTableList() for: "tx_cfcleague_profiles LEFT JOIN tx_cfcleague_teams AS t1 ON FIND_IN_SET(tx_cfcleague_profiles.uid, t1.players) " (SQL engine parse ERROR: No join field found in parseFromTables()!: near "FIND_IN_SET(tx_cfcleague_profiles.uid, t1.players)")

und ich kann dann nicht einmal einen neuen Spielplan erzeugen.

Habe danach die Extention komplett deinstalliert und die vorherige Version wieder installiert. Danach bekomme ich immer wieder die Fehlermeldung:
DBAL fatal error: No handler found in handler_getFromTableList() for: "tx_cfcleague_profiles LEFT JOIN tx_cfcleague_teams AS t1 ON FIND_IN_SET(tx_cfcleague_profiles.uid, t1.players) " (SQL engine parse ERROR: No join field found in parseFromTables()!: near "FIND_IN_SET(tx_cfcleague_profiles.uid, t1.players)")

rn_base 1.8.2
cfc_league 1.2.0
cfc_league_fe 1.2.0
php 7.1
typo3 7.6.19

Irgendetwas muss ich falsch gemacht haben.

p.s. auf der Wirkumgebung funktioniert es noch einwandfrei. Habe da aber noch nicht aktuallisiert.

**Comment by digedag** [comment]
Such mal im ExtensionManager nach der Extension "Database Abstraction Layer" (dbal). Ich denke, die ist bei dir installiert. Die Exception stammt nämlich aus dieser Extension. Wenn du sie deaktivierst, sollte es wieder funktionieren. Das sollte aber unabhängig vom Update von cfc_league sein. An der Datenbankanbindung hat sich nichts geändert... Vergleich dann mal mit deiner zweiten Umgebung, ob dbal da auch installiert ist.

**Comment by entaxi57** [comment]
Hallo René,
das war der Fehler. Danke. Da vergleicht man schon und übersieht es doch. Sorry.

Werde jetzt mit der neuesten Version testen.

**Comment by entaxi57** [comment]
Hallo René,

folgendes ist mir aufgefallen
Anzeige des Uhrfeldes:
Uhr starten = z.B. 1118:54
Uhr pausieren = 00:00
Uhr stop = 0

Die Zeit wird auch nicht übernommen, wenn ich ein Tor eintragen will.

Im Liveticker werden die Tore nur angezeigt, wenn man manuell die Zeit einträgt.

**Comment by digedag** [comment]
Also das Problem mit der Übernahme der Uhrzeit kann ich reproduzieren. Das hab ich eben korrigiert. Aber die Stopuhr läuft bei mir tadellos! Werden die im Browser irgendwelche Javascript-Fehler gemeldet? Öffne mal im Chrome oder Firefox mit Ctrl-Shift-i die Dev-Konsole. Vielleicht poppt da ja irgendein Fehler auf...

**Comment by entaxi57** [comment]
Hallo René,

bei mir fängt die Stoppuhr z.Z. mit 1190:xxx an, die Uhr fängt also nicht bei 0:00 Uhr an. Egal mit welchem Browser. In der Dev-Konsole (Chrome) kann ich keinen Fehler feststellen.

Ausnahme:
[Deprecation] The Notification API may no longer be used from insecure origins. You should consider switching your application to a secure origin, such as HTTPS. See https://goo.gl/rStTGz for more details.
LoginRefresh.initialize @ LoginRefresh.js?bust=8f534958d5b82a610e255c53e492cf95b2aa22c5:476

**Comment by entaxi57** [comment]
Hallo René,
die Übernahme der Uhrzeit klappt jetzt einwandfrei. 👍

p.s. Mach dir keinen Stress mit dem anderen "Problem". 😉

**Comment by digedag** [comment]
Mich würde aber schon interessieren, wieso die Uhr falsch angezeigt wird. In deinem ersten Kommentar war die Minute bei "1118" und im zweiten Kommentar bei "1190". Richtig? Und trotzdem wird die korrekte Minute "1" übernommen? Oder wird als Minute dann auch die 1190 eingetragen?

**Comment by entaxi57** [comment]
Hallo René,

> In deinem ersten Kommentar war die Minute bei "1118" und im zweiten Kommentar bei "1190". Richtig? = Ja.

> Und trotzdem wird die korrekte Minute "1" übernommen? Oder wird als Minute dann auch die 1190 eingetragen? = Nein (1244:xx = 124 )
